### ISSUE TITLE
[Bug]: DropDown / Select goes outside the view

### ISSUE THREAD

**Issue by maelp** [radix-vue/shadcn-vue#101]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Shadcn Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

none

### Steps to reproduce

When the list is too long, the list goes outside the view, and it's not possible to scroll

### Describe the bug

<img width="231" alt="image" src="https://github.com/radix-vue/shadcn-vue/assets/15458/1858d6e5-9f2f-4486-aabb-c67eebdf17a2">

### Expected behavior

The view should stop at the border of the screen (either top, if the items are shown above the Dropdown / Select, or bottom if shown below) and it should be scrollable

**Comment by maelp** [comment]
@zernonia if you have some tips on how I could do a small patch to fix it for now until you have time to make a clean edit I'd love to see it (I'm trying to use this for something in prod haha)

**Comment by zernonia** [comment]
I think you can add `max-height` to `DropdownMenuContent.vue` as see? 😁

**Comment by maelp** [comment]
yes but I'm not sure that's sufficient: if the screen is small and the dropdown goes "outside the view" but the max-height is bigger than the available space, it is still not going to scroll... not sure if javascript is needed?

**Comment by maelp** [comment]
Like doing something with this value? https://github.com/radix-vue/radix-vue/blob/main/packages/radix-vue/src/Select/SelectItemAlignedPosition.vue#L108

**Comment by hrynevychroman** [comment]
I think that this is successfully closed in #264 🙂
